### PR TITLE
Initialize Django Rest Framework, Add Users Viewset

### PR DESCRIPTION
**Pull request by danielchen115**
Django REST Framework makes building a RESTful API a lot easier. Also, more visual representation of data (yay!)
image

### CODE REVIEW

File: accounts/viewsets.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from rest_framework.viewsets import ModelViewSet
+from .models import User
+from .serializers import UserSerializer
+
+
+class UserViewSet(ModelViewSet):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
```
Comment (brittanylau) [inline accounts/viewsets.py]: can we also add `IsAuthenticated` to the `permission_classes`?

**Review by brittanylau — APPROVED** [review]
LGTM! just one small ask below

thanks for adding the users viewset 😎

File: accounts/viewsets.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from rest_framework.viewsets import ModelViewSet
```
Comment (brittanylau) [inline accounts/viewsets.py]: we also have a views.py file created by startapp – do ya wanna remove those or put viewsets in there?

Comment (danielchen115) [reply accounts/viewsets.py]: changed to live in views.py instead!

File: accounts/viewsets.py (hunk 1 above)
Comment (danielchen115) [reply accounts/viewsets.py]: added :)

**Comment by brittanylau** [comment]
nice!! let's 🚢

**Pull request merged**